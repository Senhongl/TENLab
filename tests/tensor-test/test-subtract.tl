a = [[1.1,2.0],[3.5,4.2]];
b = [[0.1,0.3],[2.2,6.2]];
print(a-b);
print(a-0.2);