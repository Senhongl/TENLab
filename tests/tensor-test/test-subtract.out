 1.0000  1.7000
 1.3000 -2.0000
[ CPUDoubleType{2,2} ]
 0.9000  1.8000
 3.3000  4.0000
[ CPUDoubleType{2,2} ]
